GitCommands/DiffMergeTools/TortoiseGitMerge.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+
+namespace GitCommands.DiffMergeTools
+{
+    // deprecate pre-1.8 versions (ec9e3fd3438b38b65cdfc60dc419709ca8c03aff)
+    internal class TortoiseGitMerge : DiffMergeTool
+    {
+        /// <inheritdoc />
+        public override string ExeFileName => "TortoiseGitMerge.exe";
```

Comment:
This removes support for tgit < 1.8 as well as using tsvn binary (unless that is changed too)
(just noting, probably OK)

Reply:
If different versions of diff/merge tool have different exe names/different parameter sets they must be presented as different `DiffMergeTool` definitions.
This will allow tool-specific configurations and deprecation of out-dated tools in a clean way

Reply:
Should be documented in commit message

Reply:
Added to the commit message